src/index.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
+import Profilescreen from './pages/profilescreen.js'
+import Homepage from './pages/Homepage.js'
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
```

Comment:
It would work better to render the <App/> component here for a more intuitive file structure.
The <Homepage /> and <ProfileScreen /> should go inside App.js